apps/protocol-frontend-e2e/src/integration/02_waitlist.cy.ts

```diff
@@ -0,0 +1,72 @@
+/// <reference types="cypress" />
+
+const network = "goerli";
+const address: string = Cypress.env('address');
+const COOKIE: string = Cypress.env('COOKIE');
+
+const truncateTablesQuery = `
+TRUNCATE TABLE "GuildContribution" RESTART IDENTITY CASCADE;
+TRUNCATE TABLE "Contribution" RESTART IDENTITY CASCADE;
+TRUNCATE TABLE "User" RESTART IDENTITY CASCADE;
+TRUNCATE TABLE "Guild" RESTART IDENTITY CASCADE;
+`
+const insertDAOs = `
+INSERT INTO "Guild" (id, name)
+    VALUES (1,'Govrn');
+INSERT INTO "Guild" (id, name)
+    VALUES (2,'MGD');
+`
+beforeEach(()=>{
+  cy.task('queryDatabase', truncateTablesQuery)
+    .then((res) => {
+    expect(res[0].rows.length).to.equal(0);
+  });
+  cy.task('queryDatabase', insertDAOs)
+    .then((res) => {
+    expect(res.length*res[0].rowCount).to.equal(2);
+  });
+
+  cy.fixture('users.json').then((users) => {
+    this.users = users
+  });
+  
+  cy.login(network, 
+    address, 
+    COOKIE
+  );
+});
+
+afterEach(()=>{
+  cy.login(network, 
+    address, 
+    COOKIE
+  ); 
+  
+  //Now in Discord's window
+});
+
+describe("New User Login", () => {
+  it("Fill Govrn Waitlist Form", () => {
+
+    cy.wait(3000)
+    cy.get('[data-cy="create-my-profile-btn"]')
+      .click({force:true});
+
+    cy.get('input[data-testid="chakraInput-test"]') 
+      .eq(0)  
+      .type(this.users[0].username)
+      .should('have.value', this.users[0].username);
+    
+    cy.get('input[data-testid="chakraInput-test"]') 
+      .eq(1) 
+      .type(this.users[0].email)
+      .should('have.value',this.users[0].email)
+    
+    cy.get('[data-cy="join-waitlist"]')
+      .click();
+
+    cy.wait(3000); 
```

Comment:
Why are we waiting another 3000ms at the end

Reply:
The 3000ms starts after the waitlist form has been submitted. Without the 3000ms, we wouldn't visually notice the "Successfully Joined Waitlist" DialogBox,  since the page loads faster to "join our Discord" page due to the scripts in afterEach().
In real word, the user will have to spend maybe more than 3000ms, to read that dialog box, then refresh the page and
join Discord.


<img width="640" alt="Screen Shot 2022-09-13 at 11 23 51 AM" src="https://user-images.githubusercontent.com/84946242/189981126-8c2a73bd-e1e4-4adc-8348-e40d7ce4dc36.png">